src/js/chatWindow.spec.js

```diff
@@ -37,3 +37,30 @@ test('sanitizes input and adds valid messages to the chat window', () => {
   expect(document.getElementsByClassName('chat-window__messages')[0].lastChild)
     .toStrictEqual(expectedNode);
 });
+
+describe('testing options parameter functionality in addMessageToChatHistory', () => {
+  it('throws an error if messageType is not "incoming" or "outgoing"', () => {
+    expect(() => { addMessageToChatHistory('testing', { messageType: 'unknown' }); }).toThrowError('Message Type is incorrect');
+  });
```

Comment:
I'd consider this to be a nice integration test. It tests the functionality of the guard within the `addMessageToChatHistory` function. Nice!